next/docs/iota-sdk/docs/how-tos/accounts-and-addresses/list-outputs.mdx

```diff
@@ -13,27 +13,60 @@ keywords:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This How-To will show you how to list all outputs of an account.
+This guide will show you how to list all outputs of an account.
```

Comment:
Should this say something like "all outputs that can be unlocked by an address of a given account"?